tests/end-to-end/external-solution-integration/.ko.delve.yaml

```diff
@@ -0,0 +1,2 @@
+# optionally build your own image from Dockerfile.delve
+defaultBaseImage: docker.io/nachtmaar/delve:alpine
```

Comment:
Would it make sense to add a note about this field, `defaultBaseImage`? Let's use a placeholder for `nachtmaar` to deliberately fail for others. E.g. `<add_your_dockerhub_handle>`

Reply:
I wanted to avoid that others have to build this image, if they wanna get it running very quick. But since my docker image will not be maintained by me, I understand your point. So let's do it as you propose 👍

Reply:
We discussed again: Until we have a better solution, let's keep it as it is for simplicity. Replacing ko with docker would work, but make it more complicate 🤔